rr_platform/msg/chassis_state.msg

```diff
@@ -1,5 +1,5 @@
 # Message for defining the state of the car
 Header header           # header for when the message is published
-bool estop              # whether the estop is on or not
+bool estop_on           # whether the estop is turned on
```

Comment:
Again, what does it mean for the estop to be "on"? The description here should specify how true/false maps to the robot being enabled/disabled. (same thing for the mux_automatic variable btw.